lib/turn_utils.ex

```diff
@@ -0,0 +1,105 @@
+defmodule Membrane.TURN.Utils do
+  @moduledoc false
+
+  @numbers_and_letters Enum.concat([?0..?9, ?a..?z, ?A..?Z])
+
+  @spec generate_credentials(binary(), binary()) :: {binary(), binary()}
+  def generate_credentials(name, secret) do
+    duration =
+      DateTime.utc_now()
+      |> DateTime.to_unix()
+      |> tap(fn unix_timestamp -> unix_timestamp + 24 * 3600 end)
+
+    username = "#{duration}:#{name}"
+
+    password =
+      :crypto.mac(:hmac, :sha, secret, username)
+      |> Base.encode64()
+
+    {username, password}
+  end
+
+  @spec generate_secret() :: binary()
+  def generate_secret() do
+    symbols = '0123456789abcdef'
```

Comment:
As above. Could we link something that explains how secret is constructed.

Reply:
As above